src/ci_opam.ml

```diff
@@ -26,6 +26,7 @@ let default_pkg = "my-package"
 
 let pkg =
   let pkg = getenv_default "PACKAGE" default_pkg in
+  let pkg = if String.length pkg = 0 then default_pkg else pkg in
```

Comment:
Hm, if it's not left as empty, then `opam pin add .` fails the auto-detection of the name.  Maybe we should just make PACKAGE mandatory for now? It'll fail with:

```
No opam file found for my-package
```

at the moment anyway